cpp/tests/test_odesecirvvs.cpp

```diff
@@ -550,26 +550,21 @@ TEST(TestOdeSECIRVVS, export_time_series_init)
     ASSERT_THAT(mio::create_directory(tmp_results_dir), IsSuccess());
 
     auto num_age_groups = 6; // Data to be read requires RKI confirmed cases data age groups
-    auto model          = make_model(num_age_groups);
+    auto model = std::vector<mio::osecirvvs::Model>({make_model(num_age_groups)});
 
     // Test exporting time series
     ASSERT_THAT(mio::osecirvvs::export_input_data_county_timeseries(
-                    std::vector<mio::osecirvvs::Model>{model}, TEST_DATA_DIR, tmp_results_dir, {0}, {2020, 12, 01},
-                    std::vector<double>(size_t(num_age_groups), 1.0), 1.0, 2),
+                    model, {2020, 12, 01}, {0}, 2, std::vector<double>(size_t(num_age_groups), 1.0), 1.0, TEST_DATA_DIR,
+                    tmp_results_dir, true),
                 IsSuccess());
 
     auto data_extrapolated = mio::read_result(mio::path_join(tmp_results_dir, "Results_rki.h5"));
     ASSERT_THAT(data_extrapolated, IsSuccess());
 
     // Values were generated by the tested function export_input_data_county_timeseries;
     // can only check stability of the implementation, not correctness
-    // TODO: The following test and its data has to be reworked. In MEmilio,
-    // export_input_data_county_timeseries() is not expected to be called
-    // standalone. In contrast, it expects set_vaccinvation_data() to be called
-    // first to set the number of vaccinations. If this is not done, the three
-    // susceptible compartments differ from what is exported in the simulations.
     auto expected_results =
-        mio::read_result(mio::path_join(TEST_DATA_DIR, "export_time_series_init_osecirvvs.h5")).value();
+        mio::read_result(mio::path_join(TEST_DATA_DIR, "export_time_series_init_osecirvvs_with_vaccination.h5")).value();
```

Comment:
If the old file is not used anymore, the new file should replace the old one and not be added additionally. The question is if or why we do not need to change the values in model_initialization